core/src/rpc.rs

```diff
@@ -228,6 +232,9 @@ pub trait RpcSol {
     #[rpc(meta, name = "getTransactionCount")]
     fn get_transaction_count(&self, _: Self::Metadata) -> Result<u64>;
 
+    #[rpc(meta, name = "getSolTotalSupply")]
```

Comment:
nit: let's drop Sol, so `getTotalSupply`.  We're returning Lamports here not Sol